src/web/routes/guidance/get-page-meta-data.js

```diff
@@ -0,0 +1,28 @@
+const { compose, equals, prop } = require('ramda')
+
+const getPageForPath = (pages, path) => pages.find(hasMatchingPath(path))
+
+const hasMatchingPath = (path) => compose(equals(path), prop('path'))
+
+const getPreviousPage = (pages, index) => pages[index - 1]
+
+const getNextPage = (pages, index) => pages[index + 1]
+
+const getPageMetadata = (pages, path) => {
```

Comment:
@mikeSmith08 I think `getPageMetadata` will now need to be explicitly exported from `index.js` on your branch for your ATs to work.